python/segyio/segy.py

```diff
@@ -428,12 +428,12 @@ def header(self):
 
     @header.setter
     def header(self, val):
-        """headers macro assignment
+        """standard headers macro assignment
 
-        A convenient way for operating on all headers of a file is to use the
-        default full-file range.  It will write headers 0, 1, ..., n, but uses
-        the iteration specified by the right-hand side (i.e. can skip headers
-        etc).
+        A convenient way for operating on all standard headers of a file is to
+        use the default full-file range.  It will write headers of trace 0, 1,
```

Comment:
Not that important. It probably renders correctly, but there is a double space:
```suggestion
        use the default full-file range. It will write headers of trace 0, 1,
```